### DISCUSSION TITLE
[Feature] Disable update message in mobile app

### DISCUSSION THREAD

**Discussion by Morethanevil** [immich-app/immich#2184] [Feature Request]
### The feature

It would be nice to be able to  deactivate the message about updates in the mobile app. Every user gets it, but only the admin needs it. I would like to have an option to turn off this message.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by etnoy** [comment]
On a related note, it's very easy to hit the GitHub rate limit for the endpoint that checks the version

**Comment by noshisenpai** [comment]
Any update on this feature. It would be really helpful.

**Comment by karatektus** [comment]
I feel like the app should check the connected server version and only notify the user once thats newer.

**Reply by noshisenpai** [reply]
Me personally I would just like control over the notification period on the app. I have other people using my Immich server in my family who are not tech savvy and I get asked about the updates. I have Watchtower turned on and I update overnight which may be a different time zone than the dev when they publish. Or just the admin account get a notification on the app like the suggestion.

**Comment by KusAdama** [comment]
I wrote a broader solution proposal that takes into account storage, version info, and update notification.
It's available [here](https://github.com/immich-app/immich/discussions/25761#discussion-9412414).
It was closed as a duplicate to this, so I'm just adding it also here for additional clarity.
